parm/config/gfs/config.resources

```diff
@@ -9,7 +9,7 @@ if [[ $# -ne 1 ]]; then
     echo "Must specify an input task argument to set resource variables!"
     echo "argument can be any one of the following:"
     echo "coupled_ic aerosol_init"
-    echo "prep preplandobs"
+    echo "prep preplandobs prepiodaobs"
```

Comment:
I would suggest calling this `prepatmiodaobs` to identify the `atm` component of observation processing.  Marine obs will need ioda preparation jobs and it should be distinguishable.

Reply:
Replace `prepiodaobs` with `prepatmiodaobs`.   Done at 7af58ee3.